build-tools/packages/build-cli/src/handlers/stateHandlers.ts

```diff
@@ -9,21 +9,52 @@ import { CommandLogger } from "../logging.js";
 import { MachineState } from "../machines/index.js";
 
 /**
- * An async function that handles state. Typically this type is used via the {@link BaseStateHandler} class and its
- * subclasses.
+ * An async function that is called to handle a particular state from a jssm state machine. Typically this type is used
```

Comment:
Nit: might be nice to `{@link}` to a definition for "jssm state machine", if there is a good one.